Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,92 @@
+# Preparing development environment and first rkt build
+This is an example configuration for fresh installation of Ubuntu 16.04 GNOME.
+This doc is also a short quick start guide. For a detailed reference see [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Required software
+On fresh system install few software packages to correctly build rkt:
```

Comment:
I'm not a native speaker, but I think this could better read as "On a fresh system installation, few additional software packages are needed to correctly build rkt:"